backend/app/solr/operations.py

```diff
@@ -0,0 +1,31 @@
+"""Operations on Solr"""
+from httpx import AsyncClient, Response
+
+from app.config import SOLR_URL
+
+
+async def search(
+    client: AsyncClient,
+    collection: str,
+    *,
+    q: str,
```

Comment:
This same as above. Docstrings with params or URL to its documentations + examples